### PR TITLE
Dialog story changes

### PR DESCRIPTION
**Pull request by libertymayc**
feedback from @ivan-calderon 
Make Dialog story content configurable
Fix width of container

### CODE REVIEW

File: packages/lab/stories/dialog/dialog.stories.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { PropsWithChildren, useState } from "react";
-import { Button, StackLayout } from "@salt-ds/core";
+import { Button, capitalize, StackLayout } from "@salt-ds/core";
```
Comment (origami-z) [inline packages/lab/stories/dialog/dialog.stories.tsx:2]: Are you using `capitalize`?

File: packages/lab/stories/dialog/dialog.stories.tsx [hunk 2]
```diff
@@ -166,14 +174,8 @@ const AlertDialogTemplate: ComponentStory<typeof Dialog> = ({
...
       >
-        <DialogTitle>
-          {status![0].toUpperCase()}
-          {status!.slice(1)}
-        </DialogTitle>
-        <DialogContent>
-          A Dialog is a window that opens over the application content, focusing
-          the user's attention on a particular task or piece of information.
-        </DialogContent>
+        <DialogTitle>{title}</DialogTitle>
```
Comment (origami-z) [inline packages/lab/stories/dialog/dialog.stories.tsx:177]: If `title` defaults to `undefined`, is there a case to not render `DialogTitle`?

File: packages/lab/stories/dialog/dialog.stories.tsx (hunk 1 above)
Comment (libertymayc) [reply packages/lab/stories/dialog/dialog.stories.tsx:2]: i was but removed, will take out

File: packages/lab/stories/dialog/dialog.stories.tsx (hunk 2 above)
Comment (libertymayc) [reply packages/lab/stories/dialog/dialog.stories.tsx:177]: I didn't mean to default it to undefined (was trying to see if i can override the default props) - the default should be the args given for each story and there is always one
i think conditional rendering title and content would be a different story altogether showcasing how it looks when empty (like empty formfield with no control) which wouldnt be encouraged, so i wont add it (at least for now, but will note to Ivan, if there's use cases for no title/content)

**Review by origami-z — APPROVED** [review]

**Review by ZarkoPernar — APPROVED** [review]

**Review by alycrys — APPROVED** [review]

**Review by origami-z — APPROVED** [review]

**Pull request merged**